From b56d23a02d8dc880ce94286f51cc143a91fcc06e Mon Sep 17 00:00:00 2001
From: effects3d <effects3dee@gmail.com>
Date: Thu, 27 Aug 2026 00:36:51 +0200
Subject: [PATCH 7/9] Revert "loader: Install all fonts provided by Proton in
 the registry."

We're only using the downstream fork of Wine used in Proton, so we can't
install these fonts. This also reverts a regression with WebView2 not
functioning.

Source: https://github.com/ValveSoftware/wine/commit/31b56a9
Signed-off-by: effects3d <effects3dee@gmail.com>
---
 loader/wine.inf.in | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 1ee588c2572..5bf3c2955a8 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -540,24 +540,14 @@ HKLM,%FontSubStr%,"Times New Roman TUR,162",,"Times New Roman,162"
 HKLM,System\CurrentControlSet\Hardware Profiles\Current\Software\Fonts,"LogPixels",0x10003,0x00000060
 HKLM,%FontsNT%,"Arial (TrueType)",,"arial.ttf"
 HKLM,%FontsNT%,"Arial Bold (TrueType)",,"arialbd.ttf"
+HKLM,%FontsNT%,"Times New Roman (TrueType)",,"times.ttf"
 HKLM,%FontsNT%,"Courier New (TrueType)",,"cour.ttf"
-HKLM,%FontsNT%,"Courier New Bold (TrueType)",,"courbd.ttf"
 HKLM,%FontsNT%,"Malgun Gothic (TrueType)",,"malgun.ttf"
-HKLM,%FontsNT%,"Marlett (TrueType)",,"marlett.ttf"
 HKLM,%FontsNT%,"Microsoft Sans Serif (TrueType)",,"micross.ttf"
-HKLM,%FontsNT%,"MS Gothic (TrueType)",,"msgothic.ttc"
-HKLM,%FontsNT%,"MS PGothic (TrueType)",,"msgothic.ttc"
-HKLM,%FontsNT%,"MS UI Gothic (TrueType)",,"msgothic.ttc"
 HKLM,%FontsNT%,"Microsoft YaHei (TrueType)",,"msyh.ttf"
+HKLM,%FontsNT%,"MS Gothic (TrueType)",,"msgothic.ttc"
 HKLM,%FontsNT%,"Nirmala UI (TrueType)",,"nirmala.ttf"
 HKLM,%FontsNT%,"SimSun (TrueType)",,"simsun.ttc"
-HKLM,%FontsNT%,"NSimSun (TrueType)",,"simsun.ttc"
-HKLM,%FontsNT%,"Symbol (TrueType)",,"symbol.ttf"
-HKLM,%FontsNT%,"Tahoma (TrueType)",,"tahoma.ttf"
-HKLM,%FontsNT%,"Tahoma Bold (TrueType)",,"tahomabd.ttf"
-HKLM,%FontsNT%,"Times New Roman (TrueType)",,"times.ttf"
-HKLM,%FontsNT%,"Webdings (TrueType)",,"webdings.ttf"
-HKLM,%FontsNT%,"Wingdings (TrueType)",,"wingdings.ttf"
 HKCU,Software\Wine\Fonts\Replacements,"Palatino Linotype",,"Times New Roman"
 HKCU,Software\Wine\Fonts\Replacements,"Verdana",,"Times New Roman"
 HKCU,Software\Wine\Fonts\Replacements,"Segoe UI Semibold",,"Times New Roman"
-- 
2.55.0

